### PR TITLE
BUG: there should no 's'

### PR DESCRIPTION
**Pull request by Zeroto521**
A mistake from a tiny chaning variable.

### CODE REVIEW

**Comment by Zeroto521** [comment]
Another problem is how to fix this

```bash
Started: bash /home/runner/work/_actions/stefanzweifel/git-auto-commit-action/v4/entrypoint.sh
INPUT_REPOSITORY value: .
INPUT_STATUS_OPTIONS: 
INPUT_BRANCH value: main
From https://github.com/Flow-Launcher/docs
 * [new branch]      main       -> origin/main
Previous HEAD position was 042b692 Merge 4de70d2b791a7df226026b9a6a9181195be71ecc into 2b78e43c5e1ba768604e6757cfac44249f38abd0
Switched to a new branch 'main'
M	plugins.md
Branch 'main' set up to track remote branch 'main' from 'origin'.
INPUT_ADD_OPTIONS: 
INPUT_FILE_PATTERN: .
INPUT_COMMIT_OPTIONS: 
INPUT_COMMIT_USER_NAME: GitHub Actions
INPUT_COMMIT_USER_EMAIL: actions@github.com
INPUT_COMMIT_MESSAGE: DOC: update plugin list
INPUT_COMMIT_AUTHOR: Zeroto521 <Zeroto521@users.noreply.github.com>
[main 07384a2] DOC: update plugin list
 Author: Zeroto521 <Zeroto521@users.noreply.github.com>
 1 file changed, 33 insertions(+), 27 deletions(-)
 rewrite plugins.md (75%)
INPUT_TAGGING_MESSAGE: 
No tagging message supplied. No tag will be added.
INPUT_PUSH_OPTIONS: --force
remote: Permission to Flow-Launcher/docs.git denied to github-actions[bot].
fatal: unable to access 'https://github.com/Flow-Launcher/docs/': The requested URL returned error: 403
Error: Invalid status code: 128
    at ChildProcess.<anonymous> (/home/runner/work/_actions/stefanzweifel/git-auto-commit-action/v4/index.js:17:19)
    at ChildProcess.emit (events.js:210:5)
    at maybeClose (internal/child_process.js:1021:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:283:5) {
  code: 128
}
Error: Invalid status code: 128
    at ChildProcess.<anonymous> (/home/runner/work/_actions/stefanzweifel/git-auto-commit-action/v4/index.js:17:19)
    at ChildProcess.emit (events.js:210:5)
    at maybeClose (internal/child_process.js:1021:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:283:5)
```

File: .github/workflows/plugin-updater.yml [hunk 1]
```diff
@@ -26,8 +22,8 @@ jobs:
         env:
           GH_TOKEN: ${{ github.token }}
           REPOSITORY: "Flow-Launcher/Flow.Launcher.PluginsManifest"
-          PLUGINS_JSON: "plugins.json"
-          PLUGINS_MARKDOWN: "plugins.md"
+          PLUGIN_JSON: "plugins.json"
+          PLUGIN_MARKDOWN: "plugins.md"
```
Comment (jjw24) [inline .github/workflows/plugin-updater.yml:26]: this plugins md should be plugins.md? if it is referencing the md file in the docs repo

Comment (Zeroto521) [reply .github/workflows/plugin-updater.yml:26]: fine, another error.

**Comment by jjw24** [comment]
> Another problem is how to fix this
> 
> ```shell
> Started: bash /home/runner/work/_actions/stefanzweifel/git-auto-commit-action/v4/entrypoint.sh
> INPUT_REPOSITORY value: .
> INPUT_STATUS_OPTIONS: 
> INPUT_BRANCH value: main
> From https://github.com/Flow-Launcher/docs
>  * [new branch]      main       -> origin/main
> Previous HEAD position was 042b692 Merge 4de70d2b791a7df226026b9a6a9181195be71ecc into 2b78e43c5e1ba768604e6757cfac44249f38abd0
> Switched to a new branch 'main'
> M	plugins.md
> Branch 'main' set up to track remote branch 'main' from 'origin'.
> INPUT_ADD_OPTIONS: 
> INPUT_FILE_PATTERN: .
> INPUT_COMMIT_OPTIONS: 
> INPUT_COMMIT_USER_NAME: GitHub Actions
> INPUT_COMMIT_USER_EMAIL: actions@github.com
> INPUT_COMMIT_MESSAGE: DOC: update plugin list
> INPUT_COMMIT_AUTHOR: Zeroto521 <Zeroto521@users.noreply.github.com>
> [main 07384a2] DOC: update plugin list
>  Author: Zeroto521 <Zeroto521@users.noreply.github.com>
>  1 file changed, 33 insertions(+), 27 deletions(-)
>  rewrite plugins.md (75%)
> INPUT_TAGGING_MESSAGE: 
> No tagging message supplied. No tag will be added.
> INPUT_PUSH_OPTIONS: --force
> remote: Permission to Flow-Launcher/docs.git denied to github-actions[bot].
> fatal: unable to access 'https://github.com/Flow-Launcher/docs/': The requested URL returned error: 403
> Error: Invalid status code: 128
>     at ChildProcess.<anonymous> (/home/runner/work/_actions/stefanzweifel/git-auto-commit-action/v4/index.js:17:19)
>     at ChildProcess.emit (events.js:210:5)
>     at maybeClose (internal/child_process.js:1021:16)
>     at Process.ChildProcess._handle.onexit (internal/child_process.js:283:5) {
>   code: 128
> }
> Error: Invalid status code: 128
>     at ChildProcess.<anonymous> (/home/runner/work/_actions/stefanzweifel/git-auto-commit-action/v4/index.js:17:19)
>     at ChildProcess.emit (events.js:210:5)
>     at maybeClose (internal/child_process.js:1021:16)
>     at Process.ChildProcess._handle.onexit (internal/child_process.js:283:5)
> ```

will it allow if we use repository secrets?

File: .github/workflows/plugin-updater.yml (hunk 1 above)
Comment (Zeroto521) [reply .github/workflows/plugin-updater.yml:26]: I use variable `PLUGIN_JSON` and `PLUGIN_MARKDOWN` in script no `s`

**Comment by Zeroto521** [comment]
In my local ci project, this action would use GitHub token `${{ github.token }}` automaticly.

**Comment by Zeroto521** [comment]
I figure it out. success at here https://github.com/Zeroto521/docs/runs/2813618827?check_suite_focus=true

solution from github.com/stefanzweifel/git-auto-commit-action example

```yaml
name: php-cs-fixer

on:
  pull_request:
  push:
    branches:
      - "main"

jobs:
  php-cs-fixer:
    runs-on: ubuntu-latest

    steps:
    - uses: actions/checkout@v2
      with:
        ref: ${{ github.head_ref }}

    - name: Run php-cs-fixer
      uses: docker://oskarstark/php-cs-fixer-ga

    - uses: stefanzweifel/git-auto-commit-action@v4
      with:
        commit_message: Apply php-cs-fixer changes
```

**Review by jjw24 — APPROVED** [review]

**Comment by jjw24** [comment]
the table update didnt seem to work

**Comment by Zeroto521** [comment]
> the table update didnt seem to work

https://github.com/Flow-Launcher/docs/issues/22

**Pull request merged**